.github/workflows/crystal.yml

```diff
@@ -2,31 +2,57 @@ name: CI
 
 on:
   push:
-  workflow_dispatch:
   schedule:
     - cron: "0 6 * * 1"
 
 jobs:
-  build:
+  style:
+    runs-on: ubuntu-latest
+    container: crystallang/crystal
+    steps:
+      - uses: actions/checkout@v2
+      - name: Format
+        run: crystal tool format --check
+      - name: Lint
+        uses: crystal-ameba/github-action@v0.2.12
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
```

This is already passed by default: https://github.com/crystal-ameba/github-action/blob/d5fa90462498aec2ec9c2ac7bdbb096b58c7269b/action.yml#L8

Reply:
It appears there's also a typo in the default! (cause of https://github.com/spider-gazelle/inactive-support/runs/3073115582 failure)